### ISSUE TITLE
Add CBE Group to the GPL Licence

### ISSUE THREAD

**Issue by chriswmackey** [mostaphaRoudsari/ladybug#218]
I just noticed that the CBE group changed their license over to GPL a few months ago:
https://github.com/CenterForTheBuiltEnvironment/comfort_tool/commit/127470d9229d82b1e4a62b9013fa2744746c89b9
Therefore,the next release of the Ladybug GPL license should reference the CBE tool.

@thoyt , let me know if there is any particular reference format that Ladybig should follow.

**Comment by thoyt** [comment]
We changed the license for you, remember? :)

We don't have any preference for reference format. As long as Center For the Built Environment is in there, and maybe the url (cbe.berkeley.edu), we're more than happy.

**Comment by chriswmackey** [comment]
It's all good and your work is so great that I would have come back to realize it sooner or later.  I am going through some of your code right now to help make a web tool of my own.  It is a great resource and thank you so much for making it open.

I will be sure that he URL and the words 'Center For the Built Environment' appear in the license.

On another note, I have added some new features to the Ladybug comfort models class that calculate any missing variable of the PMV equation if you have the other 5 and another Ladybug developer, @stgeorges , has added some functions to calculate Physiological Equivalent Temperature.  If either of these sound like they would fit with the contrib folder of the CBE Tool, let me know and I can add them. I also noticed that you updated the python comfort models recently and I will be sure to follow your structure if I contribute anything.

-Stay awesome,
-Chris
